src/rsz/src/EstimateWireParasitics.cc

```diff
@@ -92,11 +92,11 @@ Resizer::layerRC(dbTechLayer *layer,
                  double &res,
                  double &cap) const
 {
-  layerRC(layer->getRoutingLevel(), corner, res, cap);
+  layerRC(layer->getNumber(), corner, res, cap);
 }
 
 void
-Resizer::layerRC(int routing_level,
+Resizer::layerRC(int layer_level,
```

Comment:
Have you verified all callers (particular in grt)?

Reply:
yeah, they all call with the pointer to the techlayer (which resolves it into the correct number), I couldn't find any instances that call based on the routing_level.

Reply:
In that case let's get rid of this one and merge it into the other overload so nobody can make a mistake

Reply:
@maliberty good suggestion, turns out there was one user of it in BufferNet (so when I compiled with it protected it would still have been available). I made BufferNet use the layerRC(dbTechLayer*).